code/items/weapons/MagnetoStick.cs

```diff
@@ -0,0 +1,313 @@
+﻿using Sandbox;
+using Sandbox.Joints;
+
+using TTTReborn.Player;
+
+// Credits to the PhysGun: https://github.com/Facepunch/sandbox/blob/master/code/tools/PhysGun.cs
+
+namespace TTTReborn.Items
+{
+    [Library("ttt_magnetostick")]
+    partial class MagnetoStick : TTTWeapon // TODO Equipment
+    {
+        public override string ViewModelPath => "weapons/rust_boneknife/v_rust_boneknife.vmdl";
```

Comment:
Do we want to add a "CanDrop" param? So that users cannot drop this item?

Reply:
Nice idea, planned this but forgot somehow :D Gonna do so in this PR